packages/session-replay-browser/src/session-replay.ts

```diff
@@ -416,13 +416,7 @@ export class SessionReplay implements AmplitudeSessionReplay {
   };
 
   getDeviceId() {
-    let identityStoreDeviceId: string | undefined;
-    if (this.config?.instanceName) {
-      const identityStore = getAnalyticsConnector(this.config.instanceName).identityStore;
-      identityStoreDeviceId = identityStore.getIdentity().deviceId;
-    }
-
-    return identityStoreDeviceId || this.identifiers?.deviceId;
+    return this.identifiers?.deviceId;
```

Comment:
does this handle changes to device ids? or i guess for plugin we don't really expect device id to change.

Reply:
Kelly and I agreed that we shouldn't expect the device id to change in the middle of the ongoing replay capture. Callers can still change it through `sessionSessionId(sessionId, deviceId)`. But that will only happen when the previous session ends.